### DISCUSSION TITLE
Reporting for Payments and Webshop Purchases

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#236] [Suggestions]
Pretty much what it says on the tin. For those of us who have LLCs/businesses attached to our game servers, it would be helpful for us to be able to pull weekly, monthly, quarterly, and yearly reports on sales through GSA. The ability to export reports as PDFs would be stellar. 

In addition, more reporting data for shop packs purchased through web tokens would also be beneficial.

**Comment by mvaessen** [comment]
Sounds like that can be useful indeed. Do you have examples (or can you create examples) showing the kind of reports you're looking for?

Ideally a document (pdf) using dummy data showing exactly how you want it. That will speed up / increase chance of implementation.

**Comment by Decollates** [comment]
I can provide an example in a little bit, just please give me some time to assemble something.

**Reply by mvaessen** [reply]
Sure, not a problem at all
